### ISSUE TITLE
Easiest way to get train and validation losses

### ISSUE THREAD

**Issue by Julius-Plehn** [Nixtla/neuralforecast#562]
Thank you all for this amazing project!
Could someone direct me in the right direction regarding the easiest way to receive the training losses directly within Python without going the PyTorch Lightning/TensorBoard route?
Thanks a lot!

**Comment by kdgutier** [comment]
Hey @Julius-Plehn,

I added the train and validation loss trajectories attributes to the methods some time ago.
https://github.com/Nixtla/neuralforecast/blob/main/neuralforecast/common/_base_windows.py#L76

I believe I might need to update their recording based on the latest Pytorch-lightning updates.

Is this what you look for?

**Comment by Julius-Plehn** [comment]
Hello @kdgutier, 
Yes, that helps me a lot! Thanks!
As I am using `cross_validation` with some step and window size the variable `valid_trajectories` is expected to be empty, correct?

Thanks for your quick reply!

**Comment by kdgutier** [comment]
I am unsure about the output of the interaction between cross_validation and the trajectories.
I have used it along with fit and predict.
